packages/ui/src/bounty/modals/ContributeFundsModal/ContributeFundsModal.tsx

```diff
@@ -156,9 +164,11 @@ export const ContributeFundsModal = () => {
         transaction={transaction}
         service={service}
         controllerAccount={controllerAccount}
-        description={t('modals.contribute.authorizeDescription', { value: formatTokenValue(amount) })}
+        description={t('modals.contribute.authorizeDescription', {
+          value: formatTokenValue(state.context.amount.toString()),
```

Comment:
`formatTokenValue` supports BNs
```suggestion
          value: formatTokenValue(state.context.amount),
```